### PR TITLE
[Makefile] Stream tqdm logs for all training jobs via `neuro exec`

### PR DESCRIPTION
**Pull request by ayushkovskiy**
Closes https://github.com/neuromation/cookiecutter-neuro-project/issues/288

### CODE REVIEW

**Comment by ayushkovskiy** [comment]
If a separate target `logs-tqdm-train` does not go, we could optionally print tqdm output for each train job if an env var `PRINT_TQDM` is set

File: {{cookiecutter.project_slug}}/requirements.txt [hunk 1]
```diff
@@ -1 +0,0 @@
-tqdm==4.32.1
```
Comment (ayushkovskiy) [inline {{cookiecutter.project_slug}}/requirements.txt]: it's installed in base image already

**Comment by mariyadavydova** [comment]
I would prefer to see the output in `make train` directly. The problem is that people almost never read the docs and thus will ignore the additional option even if we describe it accurately.

Therefore, may we show `stdout` and `stderr` from a job simultaneously?

**Comment by ayushkovskiy** [comment]
`cat` steals streams from `neuro log`.

`2>&1 | tee /tmp/output.txt`

File: tests/e2e/test_e2e_template.py [hunk 2]
```diff
@@ -478,6 +478,40 @@ def _run_make_train_test(neuro_run_timeout: int, expect_patterns: List[str]) ->
...
 
+@pytest.mark.run(order=STEP_RUN)
+def test_make_train_tqdm(env_var_preset_cpu_small: str) -> None:
+    _test_make_train_tqdm()
+
+
+@try_except_finally(f"neuro kill {MK_TRAIN_JOB}")
+def _test_make_train_tqdm() -> None:
+    py_cmd = "import time, tqdm; [time.sleep(0.01) for _ in tqdm.tqdm(range(10000))]"
+    bash_cmd = f"python -W ignore -c '{py_cmd}'"
```
Comment (ayushkovskiy) [inline tests/e2e/test_e2e_template.py]: wrong `-W ignore` here

File: {{cookiecutter.project_slug}}/Makefile [hunk 3]
```diff
@@ -85,7 +85,7 @@ NEURO?=neuro
 
 
 ifeq (${TRAIN_STREAM_LOGS}, yes)
-	_TRAIN_WAIT_START_OPTION=--wait-start
+	_TRAIN_WAIT_START_OPTION=--wait-start --detach
```
Comment (mariyadavydova) [inline {{cookiecutter.project_slug}}/Makefile]: Given this change we do not need to keep detach option here. You always want to detach your train job. 

Also, why is there an underscore before the name? Looks like we do not use such a naming scheme anywhere in Makefile (e.g. we use OPTION_GCP_CREDENTIALS). Could you please rename it to TRAIN_WAIT_START?

File: {{cookiecutter.project_slug}}/Makefile [hunk 4]
```diff
@@ -314,7 +317,7 @@ kill-train:  ### Terminate the training job (set up env var 'RUN' to specify the
 kill-train-all:  ### Terminate all training jobs you have submitted (set up env var 'RUN' to specify the training job)
 ifneq ($(wildcard $(_TRAIN_JOB_FILE)),)
 	$(NEURO) kill $$(cat $(_TRAIN_JOB_FILE) | uniq -u)
-	@echo -e "Please check that jobs listed in '$(_TRAIN_JOB_FILE)' are actually deleted.\nUse 'neuro ps' to verify."
+	@echo -e "Please check that jobs listed in '$(_TRAIN_JOB_FILE)' are actually killed and delete this file manually.\nUse 'neuro ps' to verify."
```
Comment (mariyadavydova) [inline {{cookiecutter.project_slug}}/Makefile]: Killed -> terminated

File: {{cookiecutter.project_slug}}/Makefile (hunk 3 above)
Comment (ayushkovskiy) [reply {{cookiecutter.project_slug}}/Makefile]: thanks!

File: {{cookiecutter.project_slug}}/Makefile (hunk 4 above)
Comment (ayushkovskiy) [reply {{cookiecutter.project_slug}}/Makefile]: thanks

File: Makefile [hunk 5]
```diff
@@ -30,7 +30,7 @@ test_unit:
 
 .PHONY: test_e2e_dev
 test_e2e_dev:
-	PRESET=cpu-small NEURO=$(NEURO_COMMAND)  pytest -s -v --environment=dev --tb=short tests/e2e
+	PRESET=cpu-small NEURO=$(NEURO_COMMAND)  pytest -s -v --environment=dev --tb=short tests/e2e -k "make_setup or tqdm"
```
Comment (ayushkovskiy) [inline Makefile]: DEBUGGING

File: tests/e2e/test_e2e_template.py [hunk 6]
```diff
@@ -516,7 +513,7 @@ def test_make_train_custom_command(
 ) -> None:
     cmd = env_py_command_check_gpu
     cmd = cmd.replace('"', r"\"")
-    cmd = f"'python -W ignore -c \"{cmd}\"'"
+    cmd = f"bash -c 'sleep 5 && python -W ignore -c \"{cmd}\"'"
```
Comment (ayushkovskiy) [inline tests/e2e/test_e2e_template.py]: job needs to be running to use `neuro exec`

**Pull request merged**